frappe/utils/subscription.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+import requests
+
+import frappe
+
+
+@frappe.whitelist()
+def show_banner():
+	expiry = frappe.conf.expiry
+	if expiry and frappe.conf.login_url:
```

Comment:
Can we namespace these config keys? Config keys are shared among many many apps, it's best to namespace these things to avoid conflicts. 

```json
{
  "subscription": {
    "expiry": ...
  },
  ...
}
```